src/main/scala/dpla/ingestion3/mappers/Mapper.scala

```diff
@@ -135,6 +151,22 @@ trait Mapper[T, +E] extends IngestMessageTemplates {
       case(_,_) => // do nothing
     }
   }
+
+  /**
+    * Validates the presence of at least one title and logs a
+    * missing required property error if the Seq is empty
+    *
+    * @param titles Seq[String]           Values returned from rights mapping
+    * @param providerId String            The provider's local identifier
+    * @param collector MessageCollector   Ingest message collector
+    */
+  def validateTitle(titles: ZeroToMany[String], providerId: String)
+                   (implicit collector: MessageCollector[IngestMessage]): ZeroToMany[String] = {
+    if(titles.isEmpty) {
+      collector.add(missingRequiredError(providerId, "title"))
+    }
+    titles
+  }
```

Comment:
Why validate `title` with this special `validateTitle` method rather than the generic `validateRecommendedProperty` method?  Could you document the reason in a comment?

Reply:
Title is required, I think.

Reply:
Added comments to address comments